src/layers/ImageLayer.js

```diff
@@ -149,21 +139,16 @@ export default class ImageLayer extends CompositeLayer {
       lensRadius,
       id,
       onClick,
-      onHover
+      onHover,
+      modelMatrix
     } = this.props;
     const { dtype } = loader;
     const { data, width, height, unprojectLensBounds } = this.state;
     if (!(width && height)) return null;
-    const bounds = scaleBounds({
-      width,
-      height,
-      translate,
-      scale
-    });
     return new XRLayer(this.props, {
       channelData: { data, width, height },
       pickable,
-      bounds,
+      bounds: [0, height, width, 0],
```

Comment:
ah ok, so the `bounds` now are just the image dims and we use `modelMatrix` to project that in the coordinate space?

Reply:
Exactly.  These bounds are in "model space" and the `modelMatrix` takes you to world space.  This is consistent with how the `TileLayer` does it too with the `extent` prop.  Everything is communicated in model space.